.github/workflows/ci-core.yml

```diff
@@ -88,9 +88,15 @@ jobs:
           only-new-issues: ${{ github.event.schedule == '' }} # show only new issues, unless it's a scheduled run
           allow-extra-out-format-args: true
           args: --out-format checkstyle:golangci-lint-report.xml
+      - name: Install bat
+        uses: awalsh128/cache-apt-pkgs-action@1850ee53f6e706525805321a3f2f863dcf73c962 # v1.3.0
+        with:
+          packages: bat
+        if: always()
```

Comment:
Should we always pass? Seems like we'd want to fix this immediately if we didnt have access to reading reports